ansible/roles/kubernetes/tasks/secrets.yml

```diff
@@ -39,10 +39,42 @@
   set_fact:
```

Comment:
All these changes seems outside the scope of Contiv support. If I'm wrong, please explain.